nflow-engine/src/main/java/io/nflow/engine/internal/dao/ExecutorDao.java

```diff
@@ -67,13 +67,6 @@ public ExecutorDao(SQLVariants sqlVariants, @NFlow JdbcTemplate nflowJdbcTemplat
     this.hostMaxLength = env.getProperty("nflow.executor.host.length", Integer.class, -1);
   }
 
-  private int getHostMaxLength() {
```

Comment:
Should we instead make this a global helper that takes a table name and column name?